holesky/.env.example

```diff
@@ -89,13 +94,15 @@ NODE_DB_PATH_HOST=${EIGENDA_HOME}/db
 NODE_CACHE_PATH_HOST=${USER_HOME}/eigenda-operator-setup/resources/cache
 
 # TODO: Operators need to update this to their own keys
-NODE_ECDSA_KEY_FILE_HOST=${EIGENLAYER_HOME}/operator_keys/opr.ecdsa.key.json
+# Uncomment NODE_ECDSA_KEY_FILE_HOST if your need access to ecdsa key. This is generally not recommended
+# NODE_ECDSA_KEY_FILE_HOST=${EIGENLAYER_HOME}/operator_keys/opr.ecdsa.key.json
```

Comment:
Do we still generate this key in EigenLayer cli by default?

Reply:
by default we don't generate any key - they can generate or not but - it's not them.